src/python/pants/option/global_options.py

```diff
@@ -1148,6 +1155,32 @@ class BootstrapOptions:
             """
         ),
     )
+    cache_content_behavior = EnumOption(
```

Comment:
Does this only deal with remote cache content? Would it be better named `--remote-cache-content-behavior`?

Reply:
It deals with both local and remote content.